app/models/archived_patient.rb

```diff
@@ -35,6 +35,11 @@ class ArchivedPatient < ApplicationRecord
             presence: true
   validates :appointment_date, format: /\A\d{4}-\d{1,2}-\d{1,2}\z/,
                                allow_blank: true
+  validates :procedure_cost, numericality: { only_integer: true, allow_nil: true, greater_than: 0 }
```

Comment:
my experience with accountants is that they looooove being able to zero things out as an indicator to themselves of something or other. I think for every field in all these models we can validate `only_integer: true, greater_than_or_equal_to: 0` like you are for the other fields here.

Reply:
Sounds good! Updated that and added the rest of the fields in the latest revision.